images/daemon/pkg/daemon/kata_openshift.go

```diff
@@ -400,7 +417,18 @@ func installRPMs(k *KataOpenShift) error {
 		return err
 	}
 
-	_, err = copy.Image(context.Background(), policyContext, destRef, srcRef, &copy.Options{})
+	_, err = copy.Image(context.Background(), policyContext, destRef, srcRef,
+		&copy.Options{SourceCtx: sourceCtx})
+
+	if err != nil {
+		fmt.Println("Error occured when downloading payload image:")
+		fmt.Println(err)
```

Comment:
Just for consistency with what's above. 

```suggestion
		log.Printf("Error occured when downloading payload image: %v", err)
```